packages/ui/src/components/atoms/Radio/stories/Radio.mdx

```diff
@@ -1,5 +1,5 @@
 import { Story, Canvas, ArgsTable } from '@storybook/addon-docs'
-import Radio from '../Radio'
+import { Radio } from '@faststore/components'
```

Comment:
I think we can just remove this file for now, since we already have `Checkbox.stories.mdx`

Reply:
done already.